### ISSUE TITLE
APG - Definitive tooltip design pattern

### ISSUE THREAD

**Issue by stes-acc** [w3c/aria#2002]
•	[Definitive tooltip design pattern](https://github.com/w3c/aria-practices/issues/128) – when can we expect that?

**Comment by scottaohara** [comment]
should this issue be moved to the aria-practices repo?

**Comment by stes-acc** [comment]
Can we please designate a short F2F discussion slot collecting the pros and cons of current tooltip pattern recommendation, especially for "rich" tooltips containing non-presentational content like links etc.?

This goes deep and is absolutely related to the ARIA group popup/popover role discussion which addresses such cases but is unfortunately is not yet here to stay.

**Comment by MarioBatusic** [comment]
We can discuss the nature of the "tooltip" role and not the kind of implementation. The implementation proposals and examples therefor shoul be in aria/practices.
Discussion points could be:

- Is the role tooltip a document structure or a widget?
- Which content should be alowed as accessibility children if the role is a widget?
-

**Comment by stes-acc** [comment]
Attaching document with examples why current tooltip and dialog deinitions in ARIA  are not enough to express all current use cases.
[tooltipspopoversdialogs.docx](https://github.com/w3c/aria/files/12578112/tooltipspopoversdialogs.docx)

**Comment by spectranaut** [comment]
discussed at TPAC this week, notes here: https://www.w3.org/2023/09/11-aria-minutes#t07

follow up discussion will happen Friday morning, see: https://github.com/w3c/aria/wiki/TPAC-2023-ARIA-Meetings

**Comment by spectranaut** [comment]
Discussed again: https://www.w3.org/2023/09/15-aria-minutes.html

**Comment by mcking65** [comment]
@jnurthen @spectranaut 

Are there ARIA WG actions to track here? Or, should this issue be transferred to aria-practices?

**Comment by stes-acc** [comment]
As discussed on TPAC, this should evolve into a role that allows for non-presentational content like links. So, not pure APG, also ARIA evoulution included.
